### PR TITLE
docs: add app starter guide

### PR DESCRIPTION
**Pull request by gyfchong**
## Why

We have introduced several new steps to setup Kaizen in apps

## What

An App Starter guide which aims to help consumers retrofit their existing apps. Not targeted to those who use the frontend template or next services.

### CODE REVIEW

File: storybook/constants.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ export const CATEGORIES = {
 
 export const CATEGORIES_ICON = {
   [CATEGORIES.introduction]: "👋",
+  ["Guides"]: "📚",
```
Comment (HeartSquared) [inline storybook/constants.ts:14]: I'm expecting linting to yell at you though haha

```suggestion
  Guides: "📚",
```

File: docs/pages/app-starter.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+This guide is all about how to introduce the Kaizen Design System to your app.
+
+Whilst the Kaizen Design System mainly focuses on the more atomic components in your app, some of the components have a high level of complexity requiring various global components to be present in order for them to function.
+
+<InlineNotification
+  persistent
+  type="informative"
+>
+  You won't need to apply this if you started your app with the <a href="https://github.com/cultureamp/frontend-template" target="_blank">Frontend template</a> or using <a href="https://github.com/cultureamp/frontend-services/tree/main/packages/next-services">next-services</a>.
```
Comment (HeartSquared) [inline docs/pages/app-starter.stories.mdx:16]:
```suggestion
  You won't need to apply this if you started your app with the <a href="https://github.com/cultureamp/frontend-template" target="_blank">Frontend template</a> or are using <a href="https://github.com/cultureamp/frontend-services/tree/main/packages/next-services">next-services</a>.
```

File: docs/pages/app-starter.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+Be sure to read the [KaizenProvider](/?path=/docs/components-kaizenprovider-installation--page) documentation in case your app requires any further config, but generally this should be it!
+
+
+## 2. Setup Tailwind CSS
+
+It is recommended that you use Tailwind for any custom CSS you may need to write outside of using Kaizen components.
+
+For full installation instructions, see the [Tailwind Getting Started](/?path=/docs/systems-tailwind-getting-started--page) page.
+
+Wether or not you are already familiar with Tailwind, you'll want to take a look at our [Utility Class References](/?path=/docs/systems-tailwind-utility-class-references-overview--page), as we have made distinct modifications to vanilla Tailwind to suit our design system.
```
Comment (HeartSquared) [inline docs/pages/app-starter.stories.mdx:39]:
```suggestion
Whether or not you are already familiar with Tailwind, you'll want to take a look at our [Utility Class References](/?path=/docs/systems-tailwind-utility-class-references-overview--page), as we have made distinct modifications to vanilla Tailwind to suit our design system.
```

File: docs/pages/app-starter.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Add the following import to the top of your `tailwind.css` file:
+
+```css
+@import "../node_modules/@kaizen/components/dist/styles.css";
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+
+Note: If for whatever reason you aren't adding Tailwind to your project, you can do this step in any global css file. Just ensure it's the only one, imported at the root of your app, after resets/normalises
```
Comment (HeartSquared) [inline docs/pages/app-starter.stories.mdx:54]:
```suggestion
Note: If for whatever reason you aren't adding Tailwind to your project, you can do this step in any global css file. Just ensure it's the only one, imported at the root of your app, after resets/normalises.
```

File: storybook/constants.ts (hunk 1 above)
Comment (gyfchong) [reply storybook/constants.ts:14]: It didn't work with just the key 😅

Comment (HeartSquared) [reply storybook/constants.ts:14]: Haha oh well! Whatever the linter likes 😆

Comment (gyfchong) [reply storybook/constants.ts]: For some reason it works now haha

**Review by HeartSquared — APPROVED** [review]
LGTM

**Pull request merged**